tutorials/build_systems/easybuild/eb_test.py

```diff
@@ -0,0 +1,30 @@
+# Copyright 2016-2021 Swiss National Supercomputing Centre (CSCS/ETH Zurich)
+# ReFrame Project Developers. See the top-level LICENSE file for details.
+#
+# SPDX-License-Identifier: BSD-3-Clause
+
+import reframe as rfm
+import reframe.utility.sanity as sn
+
+
+@rfm.simple_test
+class BZip2EBCheck(rfm.RegressionTest):
+    descr = 'Demo test using EasyBuild to build the test code'
+    valid_systems = ['*']
+    valid_prog_environs = ['builtin']
+    executable = 'bzip2'
+    executable_opts = ['--help']
+
+    @run_before('compile')
+    def setup_build_system(self):
+        self.build_system = 'EasyBuild'
```

Comment:
How about moving the `build_system` assignment to the class body and renaming this hook to `set_easibuild_options`? Either way, this is just a comment and I don't have a strong opinion on this. However, if this is changed, the spack tutorial should also be changed for consistency reasons.

Reply:
I will still keep `setup_build_system` as the hook name.